packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -152,7 +152,7 @@ export const BootPopups = (): null => {
    * Boot popup for streaks milestone
    */
   useEffect(() => {
-    if (shouldHideStreaksModal) {
+    if (shouldHideStreaksModal || !isActionsFetched) {
```

Comment:
Should it not be move above to just make `shouldHideStreaksModal` true when actions are still not fetched?

Reply:
done